### ISSUE TITLE
geojson_to_wkt fails on valid geojson object with 'geometry' key at top level

### ISSUE THREAD

**Issue by scottstanie** [sentinelsat/sentinelsat#223]
I'm trying to use the api for a simple Polygon that looks like this:
```
{
  "type": "Feature",
  "properties": {},
  "geometry": {
	"type": "Polygon",
	"coordinates": [
	  [
		[
		  -103.46099853515624,
		  30.95876857077987
		],
		[
		  -102.72491455078125,
		  30.95876857077987
		],
		[
		  -102.72491455078125,
		  31.85423130442635
		],
		[
		  -103.46099853515624,
		  31.85423130442635
		],
		[
		  -103.46099853515624,
		  30.95876857077987
		]
	  ]
	]
  }
}
```

However, the code for the `geojson_to_wkt` function fails on a key error since it does this:

```python
if 'coordinates' in geojson_obj:
    geometry = geojson_obj
else:
    geometry = geojson_obj['features'][feature_number]['geometry']
```

It seems like it should be able to handle this type of geojson, unless I've missed something in the documentation forbidding it? The first example on the homepage for geojson also fails due to this problem: http://geojson.org/

**Comment by scottstanie** [comment]
Let me know if you also would like me to add a test for this new loading geojson type.

**Comment by j08lue** [comment]
> Let me know if you also would like me to add a test for this new loading geojson type.

yes, please

**Comment by Fernerkundung** [comment]
Closed with #224
